packages/terra-switch/src/terra-dev-site/doc/switch/Switch.1.doc.mdx

```diff
@@ -0,0 +1,60 @@
+import { Badge } from 'terra-switch/package.json?dev-site-package';
+
+import DefaultSwitch from '../example/DefaultSwitch?dev-site-example';
+import SwitchWithOnChange from '../example/SwitchWithOnChange?dev-site-example';
+import DisabledSwitch from '../example/DisabledSwitch?dev-site-example';
+import SwitchWithLongLabelText from '../example/SwitchWithLongLabelText?dev-site-example';
+
+import SwitchPropsTable from 'terra-switch/src/Switch?dev-site-props-table';
+
+<Badge />
+
+# Terra Switch
+
+The terra-switch component are like their real-world counterparts, are used to indicate whether a setting is on or off and provide users with instantaneous feedback. 
+
+Switches are used to activate settings immediately, in real time. 
+Switches never require users to press a button to apply their settings. 
+A switch is a set of binary options and always has one of the options selected by default.
+
+Switch should be used for only one of these set of binary options:
+• On / Off
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-switch`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+## Peer Dependencies
+
+This component requires the following peer dependencies be installed in your app for the component to properly function.
+
+| Peer Dependency | Version |
+|-|-|
+| react | ^16.8.5 |
+| react-dom | ^16.8.5 |
+
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## Usage
+
+```jsx
+import Switch from 'terra-switch';
+```
```

Comment:
Usage section isn't really needed, the examples all show this.
```suggestion
```

Reply:
Although this might not be needed, but to keep things consistent it would be better if this was kept.
During the MDX updates, it was decided to add these to all the packages.